.github/workflows/CI.yml

```diff
@@ -57,3 +61,203 @@ jobs:
           name: testing-artifact-api
       - name: Run tests
         run: dotnet test TestWare.Samples.API.dll
+
+  test-chrome:
+    needs: build
+    name: Run WEB (Chrome) tests
+    runs-on: windows-2022
+    steps:
+      - name: Ubuntu GitHub Actions environment variables List
+        run: env
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 6.0.x
+      - uses: actions/download-artifact@v2
+        with:
+          name: testing-artifact-selenium-web
+      - name: Replace Driver at TestConfiguration.json
+        shell: pwsh
+        run: |
+          $driverPath = "C:\SeleniumWebDrivers\ChromeDriver".Replace("\","\\")

```

Comment:
Pending to verify how to use Image default environment variables
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md